File: examples/spork/working/conflicting_wildcard_changes/Expected.java
import java.util.List;

class Cls {
<<<<<<< LEFT
    public static void print(List<? super String> list) {
||||||| BASE
    public static void print(List list) {
=======
    public static void print(List<? extends String> list) {
>>>>>>> RIGHT
        System.out.println(list);
    }
}

File: examples/haskell/working/576/Expected.hs
<<<<<<< LEFT
foo :: (A') => Foo
foo = Foo
      {
        -- comment
        bar = 0
      }
||||||| BASE
foo :: (A) => Foo
foo = Foo
      {
        -- comment
        bar = 0
      }
=======
foo :: (A) => Foo
foo = Foo
      {
        bar = 0
      }
>>>>>>> RIGHT
